core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -209,12 +213,12 @@ private[spark] object JettyUtils extends Logging {
 
       override def filterServerResponseHeader(
           clientRequest: HttpServletRequest,
-          serverResponse: Response,
+          serverResponse: org.eclipse.jetty.client.Response,
           headerName: String,
           headerValue: String): String = {
         if (headerName.equalsIgnoreCase("location")) {
           val newHeader = createProxyLocationHeader(headerValue, clientRequest,
-            serverResponse.getRequest().getURI())
+            serverResponse.getRequest.getURI)
```

Comment:
Do we need this change really? Otherwise, please revert all this kind of style change.